backend/db/migrate/20250807161828_add_investment_amount_to_dividend_computation_outputs.rb

```
@@ -0,0 +1,5 @@
+class AddInvestmentAmountToDividendComputationOutputs < ActiveRecord::Migration[8.0]
```

RuboCop (1.76.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)